sourcegraph/sourcegraph
# Gate Batch Change UI based on license

Closes #49714
- Uses new batch change licensing info exposed on the JS Context (from #52044)
- Hides Batch Change nav item on OSS
- Shows Get Started page for dotcom & enterprise customers without a Batch Changes license

## Test plan
- Run `sg start oss` - observe missing batch changes nav item (route is inaccessible too)
- Run `sg start dotcom` - observe batch changes page, only the `Get started` content displays (subrotues are inaccessible)
- Mock enterprise with limited batch changes - observe limited batch change warning once executing a SSBC
